roles/kubernetes-apps/scheduler_plugins/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Scheduler Plugins | Ensure dir exists
+  file:
+    path: "{{ kube_config_dir }}/scheduler-plugins"
+    state: directory
+    owner: root
+    group: root
+    mode: 0755
+  when: inventory_hostname == groups['kube_control_plane'][0]
+  tags:
+    - scheduler_plugins
+
+- name: Scheduler Plugins | Check helm command exists
+  command: "which helm"
+  register: helm_bin
+  ignore_errors: true  # noqa ignore-errors
+  when: inventory_hostname == groups['kube_control_plane'][0]
+  tags:
+    - scheduler_plugins
+
+- name: Scheduler Plugins | Install helm
+  include_role:
+    name: kubernetes-apps/helm
+  vars:
+    helm_enabled: true
+  when:
+    - inventory_hostname == groups['kube_control_plane'][0]
+    - helm_bin.rc != 0
+  tags:
+    - scheduler_plugins
+
+- name: Scheduler Plugins | Download scheduler plugins source
+  include_tasks: "../../../download/tasks/download_file.yml"
+  vars:
+    download: "{{ download_defaults | combine(downloads.scheduler_plugins) }}"
+  when: inventory_hostname == groups['kube_control_plane'][0]
+  tags:
+    - scheduler_plugins
+
+- name: Scheduler Plugins | Create values file
+  template:
+    src: "scheduler-plugins-values.yml.j2"
+    dest: "{{ kube_config_dir }}/scheduler-plugins/scheduler-plugins-values.yml"
+    mode: 0644
+  register: scheduler_plugins_values
+  when: inventory_hostname == groups['kube_control_plane'][0]
+  tags:
+    - scheduler_plugins
+
+- name: Scheduler Plugins | Install scheduler and controller
+  command:
+    cmd: "{{ bin_dir }}/helm upgrade --install scheduler-plugins as-a-second-scheduler/ --create-namespace --namespace scheduler-plugins -f {{ scheduler_plugins_values.dest }} --wait --timeout 5m"
+    chdir: "{{ local_release_dir }}/scheduler-plugins-{{ scheduler_plugins_version.lstrip('v') }}/manifests/install/charts"
+  when: inventory_hostname == groups['kube_control_plane'][0]
+  tags:
+    - scheduler_plugins
```

Comment:
HI @tu1h 

kubespray usually does not use "helm install" to install the addons.
Usually, it copies the manifest yaml as an ansible template for add-ons. like:

- https://github.com/kubernetes-sigs/kubespray/tree/master/roles/kubernetes-apps/ingress_controller/ingress_nginx
- https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes-apps/ansible/defaults/main.yml